pkg/kubelet/config/file_linux_test.go

```diff
@@ -314,22 +319,23 @@ func watchFileChanged(watchDir bool, symlink bool, t *testing.T) {
 			}()
 
 			if watchDir {
-				NewSourceFile(dirName, hostname, 100*time.Millisecond, ch)
+				NewSourceFile(dirName, hostname, period, ch)
 			} else {
-				NewSourceFile(file, hostname, 100*time.Millisecond, ch)
+				NewSourceFile(file, hostname, period, ch)
 			}
+
+			// await fsnotify to be ready
+			time.Sleep(time.Second)
+
 			// expect an update by SourceFile.resetStoreFromPath()
 			expectUpdate(t, ch, testCase)
```

Comment:
@liggitt 
This `expectUpdate` expects an startup update, not fsnotify event.
https://github.com/kubernetes/kubernetes/blob/0ed8ee6fd7169219fbeb9bca83023a980a7a36a5/pkg/kubelet/config/file.go#L96-L99